mPowerSDK/Dashboard/APHDashboardViewController.m

```diff
@@ -833,11 +997,10 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             graphCell.tintColor = graphItem.tintColor;
             graphCell.delegate = self;
             
-            [graphView layoutSubviews];
+            [scatterGraphView layoutSubviews];
```

Comment:
`layoutSubviewsIfNeeded`. Calling `layoutSubviews` directly is a no-no.

https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIView_Class/#//apple_ref/occ/instm/UIView/layoutSubviews

"You should not call this method directly. If you want to force a layout update, call the setNeedsLayout method instead to do so prior to the next drawing update. If you want to update the layout of your views immediately, call the layoutIfNeeded method."